scaladoc/src/dotty/tools/scaladoc/renderers/Resources.scala

```diff
@@ -171,8 +171,9 @@ trait Resources(using ctx: DocContext) extends Locations, Writer:
   )
 
   def renderResource(resource: Resource): Seq[String] =
-    if resource.path.endsWith(".html") && apiPaths.contains(resource.path) then
-      report.error(s"Conflict between resource and API member for ${resource.path}. $pathsConflictResoultionMsg")
+    val normalizedPath = resource.path.replace('\\', '/')
```

Comment:
Wouldn't be `Paths.get(resource.path).toString` more elegant in that case?

Reply:
It may still return representation with `\\`